### PR TITLE
Vtouati pbi 281196 remove adh ref

### PR DESCRIPTION
**Pull request by VictoriaTouati-AVEVA**
Replace ADH with OCS.

### CODE REVIEW

**Comment by VictoriaTouati-AVEVA** [comment]
@afloresOSI & @osisoft-mbishop: Need a quick review so we can get the ADH reference replaced ASAP on the Collect data landing page, thanks.

**Review by osisoft-mbishop — APPROVED** [review]
Approved, but you have some conflicts that need resolution.

**Comment by osisoft-mbishop** [comment]
nm, you got it sorted. Looks good!

File: content/communities/share-unshare-streams.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ To share streams with other communities, follow these steps:
 
 1. Select each stream that you want to share, and then select **Share Streams**.
 
-    **Tip:** Use the **Search for streams** field to search for specific streams. Use the wildcard operator (`*`) to return results that start or end with a particular phrase. For more information, see [Search in SDS](xref:sdsSearching).
+    **Tips:** 
+    * Use the **Search for streams** field to search for specific streams. Use the wildcard operator (`*`) to return results that start or end with a particular phrase. For more information, see [Search in SDS](xref:sdsSearching).
```
Comment (afloresOSI) [inline content/communities/share-unshare-streams.md:24]: Add a blank line between the bullets.

**Review by afloresOSI — CHANGES_REQUESTED** [review]

File: content/communities/share-unshare-streams.md (hunk 1 above)
Comment (VictoriaTouati-AVEVA) [reply content/communities/share-unshare-streams.md:24]: My edit did not touch this file.

**Pull request merged**